### PR TITLE
Handled multiple case ids as error

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-825

### Change description ###
Handled multiple case ids as error

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CreateCaseCallbackTest.java [hunk 1]
```diff
@@ -205,6 +206,18 @@ void should_respond_with_relevant_errors_when_create_case_callback_body_misses_c
...
 
+    @Test
+    void should_respond_with_relevant_error_when_multiple_cases_exist_for_given_exception_record() {
+        setUpCcdSearchResult(getCcdResponseBody("search-result-multiple.json"));
+
+        postWithBody(getRequestBody("valid-exception.json"))
+            .statusCode(INTERNAL_SERVER_ERROR.value())
+            .body("message",
+                // 1539007368674134 - case id from valid-exception.json
+                // 354, 456 - case ids from search-result-multiple.json
```
Comment (rkondratowicz) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CreateCaseCallbackTest.java]: 👍

File: src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackServiceTest.java [hunk 2]
```diff
@@ -333,25 +334,20 @@ void should_return_existing_case_if_it_exists_in_ccd_for_a_given_exception_recor
 
     @Test
     void should_return_error_if_multiple_cases_exist_in_ccd_for_a_given_exception_record() throws Exception {
```
Comment (rkondratowicz) [inline src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackServiceTest.java]: test name could be updated

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CreateCaseCallbackTest.java [hunk 3]
```diff
@@ -51,7 +52,7 @@
     @Test
     void should_create_case_if_classification_new_application_with_documents_and_ocr_data() throws IOException {
         setUpTransformation(getTransformationResponseBody("ok-no-warnings.json"));
-        setUpCcdSearchEmptyResult(getCcdResponseBody("search-result-empty.json"));
+        setUpCcdSearchResult(getCcdResponseBody("search-result-empty.json"));
```
Comment (lgonczar) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CreateCaseCallbackTest.java]: Codacy Issue found: [The String literal "search-result-empty.json" appears 7 times in this file; the first occurrence is on line 55](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4473817)

**Comment by lgonczar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4473817&bid=15104418)

**Pull request merged**